File: LAB_5_C#_C++/Line.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OOP_LAB5_CS
{
    class Line
    {
        protected Point begin;
        protected Point end;

        public Line(double begin_x, double begin_y, double end_x, double end_y)
        {
            this.begin.x = begin_x;
            this.begin.y = begin_y;

            this.end.x = end_x;
            this.end.y = end_y;
        }

        public double Length()
        {
            return Math.Sqrt(Math.Pow(end.x - begin.x, 2) + Math.Pow(end.y - begin.y, 2));
        }
    }
}

File: LAB_5_C#_C++/Line.cpp
#include <Math.h>
#include "Points.h"
#include "Line.h"

Line::Line(double begin_x, double begin_y, double end_x, double end_y)
{
	begin.x = begin_x;
	begin.y = begin_y;

	end.x = end_x;
	end.y = end_y;
};

double Line::Length()
{
	return sqrt(pow(end.x - begin.x, 2) + pow(end.y - begin.y, 2));
};